metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_classes.py

```diff
@@ -218,6 +218,7 @@ class Report:
     webUrl: Optional[str]
     embedUrl: str
     description: str
+    dataset_id: Optional[str]
     dataset: Optional["PowerBIDataset"]
```

Comment:
how do dataset_id and dataset interact? is one filled initially, and then the other populated later?

would be good to add some comments to make this more clear